mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -275,12 +275,20 @@ def _post_filter(self, updated_stacks: list[ImageStack], task):
                     # otherwise check with user which one to keep
                     if self.view.safeApply.isChecked():
                         use_new_data = self._wait_for_stack_choice(stack, stack.id)
-                    # if the stack that was kept happened to have a proj180 stack - then apply the filter to that too
-                    if stack.has_proj180deg() and use_new_data and not self.applying_to_all:
-                        # Apply to proj180 synchronously - this function is already running async
-                        # and running another async instance causes a race condition in the parallel module
-                        # where the shared data can be removed in the middle of the operation of another operation
-                        self._do_apply_filter_sync([stack.proj180deg])
+
+                    if use_new_data and not self.applying_to_all:
+                        if self.is_a_proj180deg(stack):
+                            if self.model.selected_filter.allow_for_180_projection:
+                                self._do_apply_filter_sync([stack.proj180deg])
+                            else:
+                                if task.progress:
+                                    task.progress("Skipping normalization for the 180-degree stack.")
+                                continue
```

Comment:
This just needs the `if self.model.selected_filter.allow_for_180_projection:` check added. The `self.is_a_proj180deg(stack):` is breaking the tests (makes this never run). Also the `task.progress` message is not needed.